changelog/unreleased/change-use-memory-storage

```diff
@@ -0,0 +1,7 @@
+Change: Keep user token in the memory storage
```

Comment:
IMO still an enhancement, because it's configurable, so it's not breaking anything.

Reply:
Right, before had it as forced type... will fix it